README.md

```diff
@@ -1,14 +1,64 @@
+
+## Choice Chips
+
+Blazor Chips can be grouped together as a collection of choices. Choice chips allow users to select from available options.
+
+## Filter Chips
+
+Filter chips are a collection of Blazor chips that allow users to filter the view items based on a specified parameter.
+
+## Input Chips
+
+Input chips allow Blazor chips to be added or removed. A chips can be created from text provided by the user in an input box or deleted as needed.
+
+## Related links
+[Learn More about Blazor Chips](https://www.syncfusion.com/blazor-components/blazor-chips?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/chip/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[View Demos](https://blazor.syncfusion.com/demos/chips/default-functionalities?theme=bootstrap5?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Online examples](https://blazor.syncfusion.com/demos/chips/default-functionalities?theme=bootstrap5?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Knowledge Base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+### About Syncfusion Blazor Components
+The Syncfusion's Blazor components library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Chips, we provide popular Blazor Components such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-chips-github), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-chips-github), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-chips-github), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-chips-github), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-split-chips-github).
```

Comment:
add link to this word -  Blazor components

Reply:
resolved